coala_use.md

```diff
@@ -0,0 +1,29 @@
+#Coala
+
+This is a guide on how to use coala-bears for this project
+
+This project consists of a .coafile which basically has all the guidelines
+that needs to be followed when working with coala.It can modified.
+
+In order to specify the files to analyze, you can use the --files argument. 
```

Comment:
code highlight `--files`

Reply:
btw we don't need `--files` argument

Reply:
you just need to run coala, and since `.coafile` is already present, settings would be supplied from there.

So instead of this section, you should add instructions to install coala using pip, and use it inside the project directory!

Reply:
okay will do